evap/static/scss/components/_badge.scss

```diff
@@ -19,17 +19,14 @@
 %badge {
     border-radius: $badge-border-radius;
     text-align: center;
-    width: 3rem;
+    min-width: 4rem;
+    padding: 0 0.3rem;
```

Comment:
Do we intend to force 0 padding along one axis? Otherwise, we can just use padding-block / padding-inline